app/js/lib/foxbox/db.js

```diff
@@ -25,31 +18,44 @@ const DB_VERSION = 1;
 
 const DB_SERVICE_STORE = 'services';
 
+class FoxboxIndexedDbStorage extends IndexedDbStorage {
```

Comment:
I'd rather call it something like `FoxBoxDbStorage` as it isn't always backed by IndexedDB.
What do you think?